packages/paypal-commerce-integration/src/paypal-commerce-credit-card/paypal-commerce-credit-cards-strategy.spec.ts

```diff
@@ -282,115 +243,9 @@ describe('PayPalCommerceCreditCardsPaymentStrategy', () => {
-                        'font-weight': '400',
-                    },
-                    '.invalid': {
-                        color: 'red',
-                        'font-family': 'bigFont',
-                        'font-size': '14px',
-                        'font-weight': '400',
-                    },
-                    ':focus': {
-                        color: 'black',
-                        'font-family': 'bigFont',
-                        'font-size': '14px',
-                        'font-weight': '400',
-                    },
-                },
-                paymentsSDK: true,
-                createOrder: expect.any(Function),
-            });
-        });
-
-        it('sets hosted form events if related callbacks were provided', async () => {
-            const hostedFieldOn = jest.fn();
-
-            jest.spyOn(paypalSdk.HostedFields, 'render').mockImplementation(() =>
-                Promise.resolve({
-                    on: hostedFieldOn,
-                }),
-            );
-
-            const initializationOptionsWithStyles = {
-                methodId,
-                paypalcommercecreditcards: {
-                    form: {
-                        fields: paypalCommerceCreditCardsOptions.form.fields,
-                        onBlur: jest.fn(),
-                        onFocus: jest.fn(),
-                        onEnter: jest.fn(),
-                        onCardTypeChange: jest.fn(),
-                        onValidate: jest.fn(),
-                    },
-                },
-            };
-
-            await strategy.initialize(initializationOptionsWithStyles);
-
-            expect(hostedFieldOn).toHaveBeenCalledWith('blur', expect.any(Function));
-            expect(hostedFieldOn).toHaveBeenCalledWith('focus', expect.any(Function));
-            expect(hostedFieldOn).toHaveBeenCalledWith('inputSubmitRequest', expect.any(Function));
-            expect(hostedFieldOn).toHaveBeenCalledWith('cardTypeChange', expect.any(Function));
-            expect(hostedFieldOn).toHaveBeenCalledWith('validityChange', expect.any(Function));
-        });
-
-        it('throws an error if an element with provided card name container id is not defined', async () => {
-            document.body.removeChild(paypalCardNameFieldElement);
-
-            try {
-                await strategy.initialize(initializationOptions);
-            } catch (error) {
-                expect(error).toBeInstanceOf(InvalidArgumentError);
-            }
-        });
-
-        it('renders card name field for credit card form', async () => {
-            await strategy.initialize(initializationOptions);
-
-            const cardNameContainer = document.getElementById(paypalCardNameFieldContainerId);
-            const cardNameField = cardNameContainer?.getElementsByTagName('input')[0];
-
-            expect(cardNameField).toBeDefined();
-        });
-
-        it('does not render card name field for stored credit card form', async () => {
```

Comment:
Releted to this one https://github.com/bigcommerce/checkout-sdk-js/pull/2261#discussion_r1407631328

Reply:
Replied here https://github.com/bigcommerce/checkout-sdk-js/pull/2261#discussion_r1407631328